packages/@okta/vuepress-site/docs/guides/sharing-cert/main/index.md

```diff
@@ -0,0 +1,254 @@
+---
+title: Share application key credentials for IdPs across Apps
+excerpt: How to share application key credentials between Apps
+layout: Guides
+---
+
+In this article you will learn how to share application key credentials across multiple applications so that they can be authorized using the same identity provider.
```

Comment:
```suggestion
In this article you learn how to share application key credentials across multiple applications so that they can be authorized using the same Identity Provider.
```